src/tools/demo.ts

```diff
@@ -85,10 +87,21 @@ function remove(contents: string): string {
   return result
 }
 
+/**
+ * Perform replace on all types of @demo markup
+ * @param contents The file contents as a string
+ * @return The file contents with all @demo related CommentType removed
+ */
+function sanitize(contents: string): string {
```

Comment:
Maybe rename to `sanitizeMarkup` ?

Reply:
I'm okay with `sanitize` as long as the comment is there.